draft-thoji-scone-protocol.md

```diff
@@ -197,12 +207,12 @@ when a network element detects a flow using more bandwidth than advertised via
 SCONE, it might switch to applying its policies for non-SCONE flows, using
 congestion control signals.
 
-The time and scope over which a rate limit applies is not specified.  The
-effective rate limit might change without being signaled.  The signaled limit
-can be assumed to apply to the flow of packets on the same UDP address tuple for
-the duration of that flow.  Rate limiting policies often apply on the level of a
-device or subscription, but endpoints cannot assume that this is the case.  A
-separate signal can be sent for each flow.
+The time and scope over which throughput advice applies is not specified.  The
+effective throughput advice might change without being signaled.  The signaled
+advice can be assumed to apply to the flow of packets on the same UDP address 
+tuple for the duration of that flow.  For rate limiting networks, rate limiting
+policies often apply on the level of a device or subscription, but endpoints
+cannot assume that this is the case.  A separate signal can be sent for each flow.
```

Comment:
The tentative position thus far is that we will be specifying a time period, even if we do not define a scope.  I'd rather not touch this text just now if it is changing.

Reply:
> +1. This PR should be limited to replacing "rate limit" by "Throughput advice".

I think it can also include Martin's text that says rate limiting is not the only use (line 83-86 in the PR), and together those would cover my motive for raising the PR

Reply:
I'm OK with this generally now.  However, this no longer makes sense as a statement:

> The effective throughput advice might change without being signaled.

Should we drop that line?  Or is there a different statement for this?  I'm leaning toward cutting it.

Reply:
Isn't it still the case that when the advice changes while the endpoints is in quiescence, the endpoints would end up sending data without being signaled?

Reply:
> would removing 'advice' here cover it, i.e. 'The effective throughput might change without being signaled'?
To me that misses the quiescence point that @kazuho makes, since "effective throughput" sounds like something active that you measure.

Perhaps rephrasing it to something like:

“Network conditions and rate-limit policies can change in ways that make previously signaled advice obsolete, and there are no guarantees that updated advice will be sent.”